repl.rkt

```diff
@@ -40,13 +40,21 @@
       (display (~a (fn row col) #:width width #:align 'right)))
     (newline)))
 
-(define-syntax-rule (list-find-match l pattern body ...)
-  (let loop ([l l])
-    (match l
-      [(cons pattern rest)
-       body ...]
-      [(cons _ rest) (loop rest)]
-      ['() ""])))
+(define (lookup-execution execs
```

Comment:
The function looks right, not the prettiest but whatever, it's part of the REPL. I assume you've done the rewrite to it correctly.